_posts/2016/2016-07-26-xconf-2016.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "XConf 2016 Hamburg"
+date: "2016-07-26 00:00:00"
+image: blog-header/lego.jpeg
+categories: events
+authors: ["Oleksandr V", "Renato"]
+---
+
+Not so long time ago, in one of so few sunny days in Hamburg I have had an opportunity to visit
+[XConf](https://info.thoughtworks.com/Xconf-hamburg-2016.html). Here in ePages, developers have a great support by the company if they are interested in self-education.
+
+This yearly event took place on 12th of July in a cozy place [Betahaus](http://hamburg.betahaus.de/) in format of one day conference covering "broad range of technology topics".
+
+Thanks to the great job made by [ThoughtWorks](https://www.thoughtworks.com/) I have enjoyed time participating it.
```

Comment:
Thanks to the great job of [ThoughtWorks](https://www.thoughtworks.com/) me and my colleague Renato had a blast.